interface/patient_file/report/custom_report.php

```diff
@@ -46,7 +46,7 @@
 $PDF_OUTPUT = empty($_POST['pdf']) ? 0 : intval($_POST['pdf']);
 
 if ($PDF_OUTPUT) {
-  require_once("$srcdir/html2pdf/vendor/autoload.php");
+  require_once($GLOBALS['modules_dir'] . "/html2pdf/vendor/autoload.php");
   $pdf = new HTML2PDF ($GLOBALS['pdf_layout'],
```

Comment:
The change to add Ray's statements had that change. Is it not still in the Codebase?